ui2/src/pages/network/Actions.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Tile from '../../components/Tile/Tile';
+import { Button } from 'semantic-ui-react';
+import { NavLink } from 'react-router-dom';
+
+type Action = {
+  path: string;
+  label: string;
+};
+
+type Params = {
+  actions: Action[];
+};
+
+export const ActionTile: React.FC<Params> = ({ actions }) => {
+  return (
+    <Tile header={<h4>Actions</h4>}>
+      <div className="action-row">
+        <Button.Group>
+          {actions.map(a => (
+            <NavLink to={a.path}>
```

Comment:
This should probably be a `<Link>` rather than a `<NavLink>`. I've always been prone to favor NavLinks in the past, but they're not necessary unless you need the CSS classname for active styling :grin: